lib/krane/resource_deployer.rb

```diff
@@ -128,17 +135,15 @@ def deploy_resources(resources, prune: false, verify:, record_summary: true)
       end
     end
 
-    def apply_all(resources, prune)
+    def apply_all(resources, prune, dry_run: false)
```

Comment:
Hmm, this is going to ruin the `apply_all` measurements since StatsD will now record for both dry run and actual deployments :/ I think I'm going to have to get rid of the `measure_method` helper and deal with that manually